weeks/week05/modules.md

```diff
@@ -1,3 +1,150 @@
 # Section 1: Modules
 
-Coming soon.
+Python modules and Python packages (which will be covered in the next section), two mechanisms that facilitate modular programming.
```

Comment:
I think it can be expanded to define modular programming and why it is essential for collaboration in larger codebases and facilitates debugging (and provide examples).